src/system.rs

```diff
@@ -121,6 +121,11 @@ pub trait RunNow<'a> {
 
     /// Sets up `World` for a later call to `run_now`.
     fn setup(&mut self, world: &mut World);
+
+    /// Performs clean up that requires resources from the `World`.
+    /// This commonly removes components from `world` which depend on external
+    /// resources.
+    fn dispose(&mut self, world: &mut World);
```

Comment:
prob want a default impl here and an impl in the blanket? This should prob also reduce the amount of breaking thatll happen. I assume its safe to just do nothing here as a default?

Reply:
I'm not sure. The `RunNow` API is pretty much for consuming only, not for implementing.